MdePkg/Include/Protocol/Cpu2.h

```diff
@@ -0,0 +1,51 @@
+/** @file
+  CPU2 Protocol
+
+  This code abstracts the DXE core from processor implementation details.
+
+  Copyright (c) Microsoft Corporation. All rights reserved.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+  This program and the accompanying materials
+  are licensed and made available under the terms and conditions of the BSD License
+  which accompanies this distribution.  The full text of the license may be found at
+  http://opensource.org/licenses/bsd-license.php
+
+  THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
+  WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
+
+**/
+
+#ifndef __PROTOCOL_CPU2_H__

```

Comment:
Header guard should not start with underscores [doc](https://tianocore-docs.github.io/edk2-CCodingStandardsSpecification/release-2.20/5_source_files/53_include_files.html#53-include-files)

Unless this is an exception @apop5  ?

Reply:
New header files should not have preceding underscores.

Reply:
Updated the header file to follow the doc.